### PR TITLE
Task-2: BE: Setup project

### PR DESCRIPTION
**Pull request by denys-rybachok**
add yup, config aliases, prettier, add main folders

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,10 @@
     "@nestjs/common": "^10.0.0",
     "@nestjs/core": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
+    "nestjs-yup": "^1.0.13",
```
Comment (VYaniv) [inline package.json:26]: What the package is it? Attach the link please.

File: tsconfig.json [hunk 2]
```diff
@@ -10,6 +10,13 @@
     "sourceMap": true,
     "outDir": "./dist",
     "baseUrl": "./",
+    "paths": {
+      "@app/*": ["src/*"],
+      "@utils/*": ["src/utils/*"],
+      "@constants/*": ["src/constants/*"],
+      "@entities/*": ["src/entities/*"],
```
Comment (VYaniv) [inline tsconfig.json:17]: Probably it will be better to keep entity in the folder with the module, but it is up to you.

File: package.json (hunk 1 above)
Comment (denys-rybachok) [reply package.json:26]: I planned to use it with yup for validation. But now I found another solution. I created validationPipe for Yup, so I will remove this package.

Comment (VYaniv) [reply package.json:26]: Please remove and don`t use different unpopular libraries, they can be unstable.

**Review by VYaniv — APPROVED** [review]

**Comment by VYaniv** [comment]
After you merge and pull these changes into the next PR I will can proceed with it.

**Pull request merged**